doc/source/whatsnew/v0.24.0.txt

```diff
@@ -238,6 +238,40 @@ For situations where you need an ``ndarray`` of ``Interval`` objects, use
    np.asarray(idx)
    idx.values.astype(object)
 
+
+.. _whatsnew_0240.api.types.is_scalar:
+
+Support for PEP 3141 numbers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `is_scalar` function now returns True when a `Number` or `Fraction` is passed.
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+   In [1]: pandas.api.types.is_scalar(fractions.Fraction(1))
+   Out[1]:
+   False
+
+   In [2]: pandas.api.types.is_scalar(numbers.Number(1))
+   Out[2]:
+   False
+
+New Behavior:
+
```

Comment:
you don't need this whole sub-section ,this is a minor function, however adding these examples to the doc-string would be ok.

Reply:
the is_scalar doc-strings, though I dont' think we have an Examples section, if you can add one with some examples would be great, have a look at ``infer_dtype`` for some inspiration on a good doc-string

Reply:
Will do, @jreback .